net/shorewall6-lite/Makefile

```diff
@@ -28,7 +28,7 @@ include $(INCLUDE_DIR)/package.mk
 define Package/shorewall6-lite
     SECTION:=net
     CATEGORY:=Network
-    DEPENDS:=+ip +iptables6 +shorewall-core
+    DEPENDS:=+ip +iptables +shorewall-core
```

Comment:
ip6tables does not exists. iptables is ip4 and ip6. Makefle iptables says:( Netfilter (IPv4/IPv6) )